dockerHashing/src/main/scala/cromwell/docker/DockerImageIdentifier.scala

```diff
@@ -42,7 +42,7 @@ object DockerImageIdentifier {
           :                                     # Tag separator. ':' is followed by a tag
        
           (                                     # Begin capturing group for reference 
-            [A-Za-z0-9]+(?:[.:_-][A-Za-z0-9]+)* # Reference
+            [A-Za-z0-9]+(?:[.:_\-A-Za-z0-9]+)*  # Reference
```

Comment:
if you put the dash first in the character class you shouldn't have to escape it:

```
[A-Za-z0-9]+(?:[.:_\-A-Za-z0-9]+)* // original
[A-Za-z0-9]+(?:[-.:_A-Za-z0-9]+)* // proposed
```

Reply:
Also we should probably add a case to the existing list in `DockerImageIdentifierSpec` that confirms this works as expected.